.github/workflows/ci.yml

```diff
@@ -136,21 +136,66 @@ jobs:
         run: |
           pytest -v tests
 
-  # GitHub allows you to set checks as required before PRs can be merged, which is annoying to do
-  # for matrix jobs like the one above which have six actual jobs. Instead we have a summary job
-  # like this, which we mark as required.
-  # Copied in part from:
-  # https://github.com/orgs/community/discussions/26822#discussioncomment-3305794
-  ci-report-status:
-    name: report CI status
-    needs: test
+  pypi:
+    name: Build and publish Python 🐍 distributions 📦 to PyPI
```

Comment:
since we've added this here, do we still need the [`publish.yml`](https://github.com/chemprop/chemprop/blob/main/.github/workflows/publish.yml) file?

Reply:
We do not, but we could also delete this and keep the publish file instead if we want. Your call

Reply:
doesn't matter to me. I suggest deleting the publish file since that looks more outdated. Should be good to merge after removing that

Reply:
Done